packet/flatcar-linux/kubernetes/workers/cl/install.yaml.tmpl

```diff
@@ -88,14 +88,59 @@ storage:
             mkfs.ext4 /dev/md/node-local-storage
           }
 
+          function create_disk_specific_data_raid() {
+            # Ignore the disk on which Linux is installed when selecting disks for RAID.
+            local osdisk="$1"
+            local major_numbers="$2"
+            # for hdd value is 1 and for ssd value is 0
+            local disk_type="$3"
+            # RAID device path on disk
+            local device_path="$4"
+
+            # Select disks for RAID.
+            local disks=$(lsblk -lnpd -o name,rota -I "$${major_numbers}" \
+              | grep "$${disk_type}" \
+              | sort -h -k 4,4 \
+              | grep -vE "^$${osdisk} " \
+              | awk '{x=$1 " " x;} END{print x}'
```

Comment:
Maybe `tr \\n " "`?

Reply:
This was copied directly from the other function that is used to create raid for all the devices!